### PR TITLE
Dodanie klasy dla zbioru danych oraz dataloadera

### PR DESCRIPTION
**Pull request by nuklearbanana**
Klasa zawarta w dataset_class.py korzysta z funkcji prep_data oraz give_label w celu stwoerzenia wygodnego dla pytorcha dataloadera.  Dla naszego zbioru danych (w przykładzie dwa foldery '0_0' i '0_1') bierze podaną wartość np. 1000 pierwszych obrazków: 

'pre_dataset = PixelDataset(folder_names=["0_0","0_1"],labels=[0,1], start=0, stop=1000)'

modyfikuje zbiór danych do formatu pożądanego przez dataloader. Po tej operacji można już go tworzyć, dorobić batche, pomieszać dane itp.. 

'dataloader = DataLoader(pre_dataset, batch_size=64, shuffle=True)'

### CODE REVIEW

**Comment by nuklearbanana** [comment]
Trzeba jeszcze zmienić w funkcji 'prep_data()' normalizację: 
'img_float = pixels.astype(np.float32)/ 255 na 'img_float = pixels.astype(np.float32)/127.5-1.0'
Czyli zamiast przedziału [0;1] będzie [-1;1]
